src/main/java/seedu/hirehub/storage/JsonSerializableJobList.java

```diff
@@ -0,0 +1,60 @@
+package seedu.hirehub.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import seedu.hirehub.commons.exceptions.IllegalValueException;
+import seedu.hirehub.model.job.Job;
+import seedu.hirehub.model.job.UniqueJobList;
+
+/**
+ * An immutable job list that is serializable to JSON format.
+ */
+@JsonRootName(value = "jobs")
```

Comment:
Should jobs be job list (also in other places such as method names)?

Reply:
I am not quite of the breaking changes this will cause so I will stick to jobs for now within this file